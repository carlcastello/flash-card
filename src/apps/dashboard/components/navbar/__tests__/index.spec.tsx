import React from 'react';
import { shallow } from "enzyme";
import { Navbar } from "../index";
import toJson from 'enzyme-to-json';

describe("NavBar", () => {
  let wrapper: any;

  beforeEach(() => {
    wrapper = shallow(
      <Navbar
        classes={{
          link: "link"
        }}/>
    )
  });
  
  it("Renders the NavBar", () => { 
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});