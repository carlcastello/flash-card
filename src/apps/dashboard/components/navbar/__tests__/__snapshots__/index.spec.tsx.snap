// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar Renders the NavBar 1`] = `
<WithStyles(ForwardRef(AppBar))
  position="static"
>
  <WithStyles(ForwardRef(Toolbar))>
    <WithStyles(ForwardRef(Typography))
      variant="h5"
    >
      <Link
        className="link"
        to=""
      >
        Flashcard
      </Link>
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Toolbar))>
</WithStyles(ForwardRef(AppBar))>
`;
